### PR TITLE
feat(clerk-js): Support transferable prop on SignIn

### PR DESCRIPTION
**Pull request by BRKalow**
## Description

fixes SDKI-531

Support a `transferable` prop on `<SignIn>`, as well as on `Clerk.handleRedirectCallback()`. When `false`, a sign in attempt using SSO will not automatically transfer to sign up if no account exists.

## Checklist

- [x] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Comment by BRKalow** [comment]
!snapshot

**Comment by BRKalow** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @BRKalow - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/astro | 1.0.5-snapshot.v97b12b9 |
| @clerk/backend | 1.5.0-snapshot.v97b12b9 |
| @clerk/chrome-extension | 1.1.8-snapshot.v97b12b9 |
| @clerk/clerk-js | 5.11.0-snapshot.v97b12b9 |
| @clerk/elements | 0.10.8-snapshot.v97b12b9 |
| @clerk/clerk-expo | 2.0.1-snapshot.v97b12b9 |
| @clerk/express | 0.0.22-snapshot.v97b12b9 |
| @clerk/fastify | 1.0.24-snapshot.v97b12b9 |
| gatsby-plugin-clerk | 5.0.0-beta.45 |
| @clerk/localizations | 2.5.3-snapshot.v97b12b9 |
| @clerk/nextjs | 5.2.9-snapshot.v97b12b9 |
| @clerk/clerk-react | 5.3.0-snapshot.v97b12b9 |
| @clerk/remix | 4.2.8-snapshot.v97b12b9 |
| @clerk/clerk-sdk-node | 5.0.21-snapshot.v97b12b9 |
| @clerk/shared | 2.4.2-snapshot.v97b12b9 |
| @clerk/tanstack-start | 0.1.12-snapshot.v97b12b9 |
| @clerk/testing | 1.2.4-snapshot.v97b12b9 |
| @clerk/themes | 2.1.15-snapshot.v97b12b9 |
| @clerk/types | 4.10.0-snapshot.v97b12b9 |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/astro`
```sh
npm i @clerk/astro@1.0.5-snapshot.v97b12b9 --save-exact
```
`@clerk/backend`
```sh
npm i @clerk/backend@1.5.0-snapshot.v97b12b9 --save-exact
```
`@clerk/chrome-extension`
```sh
npm i @clerk/chrome-extension@1.1.8-snapshot.v97b12b9 --save-exact
```
`@clerk/clerk-js`
```sh
npm i @clerk/clerk-js@5.11.0-snapshot.v97b12b9 --save-exact
```
`@clerk/elements`
```sh
npm i @clerk/elements@0.10.8-snapshot.v97b12b9 --save-exact
```
`@clerk/clerk-expo`
```sh
npm i @clerk/clerk-expo@2.0.1-snapshot.v97b12b9 --save-exact
```
`@clerk/express`
```sh
npm i @clerk/express@0.0.22-snapshot.v97b12b9 --save-exact
```
`@clerk/fastify`
```sh
npm i @clerk/fastify@1.0.24-snapshot.v97b12b9 --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.0-beta.45 --save-exact
```
`@clerk/localizations`
```sh
npm i @clerk/localizations@2.5.3-snapshot.v97b12b9 --save-exact
```
`@clerk/nextjs`
```sh
npm i @clerk/nextjs@5.2.9-snapshot.v97b12b9 --save-exact
```
`@clerk/clerk-react`
```sh
npm i @clerk/clerk-react@5.3.0-snapshot.v97b12b9 --save-exact
```
`@clerk/remix`
```sh
npm i @clerk/remix@4.2.8-snapshot.v97b12b9 --save-exact
```
`@clerk/clerk-sdk-node`
```sh
npm i @clerk/clerk-sdk-node@5.0.21-snapshot.v97b12b9 --save-exact
```
`@clerk/shared`
```sh
npm i @clerk/shared@2.4.2-snapshot.v97b12b9 --save-exact
```
`@clerk/tanstack-start`
```sh
npm i @clerk/tanstack-start@0.1.12-snapshot.v97b12b9 --save-exact
```
`@clerk/testing`
```sh
npm i @clerk/testing@1.2.4-snapshot.v97b12b9 --save-exact
```
`@clerk/themes`
```sh
npm i @clerk/themes@2.1.15-snapshot.v97b12b9 --save-exact
```
`@clerk/types`
```sh
npm i @clerk/types@4.10.0-snapshot.v97b12b9 --save-exact
```

**Review by dstaley — APPROVED** [review]

File: .changeset/gold-emus-talk.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@clerk/clerk-js": minor
+"@clerk/types": minor
+"@clerk/clerk-react": minor
+---
+
+Introduce `transferable` prop for `<SignIn />` to disable the automatic transfer of a sign in attempt to a sign up.
```
Comment (BRKalow) [inline .changeset/gold-emus-talk.md:7]:
```suggestion
Introduce `transferable` prop for `<SignIn />` to disable the automatic transfer of an SSO attempt with a non-existent account to a sign up.
```

**Pull request merged**